<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="serival_label">
           <property name="text">
            <string>串口号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="servial_combo_">
           <item>
            <property name="text">
             <string>COM1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM4</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>云台地址</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="addr_spin_box_"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>波特率</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="bit_combo_">
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>56000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128000</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>预置点</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="presetpoint_spin_box_"/>
         </item>
         <item>
          <widget class="QPushButton" name="call_presetpoint_btn_">
           <property name="text">
            <string>启用</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="set_presetpoint_btn_">
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_presetpoint_btn_">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="up_btn_">
         <property name="text">
          <string>↑</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="left_btn_">
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="right_btn_">
         <property name="text">
          <string>→</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="down_btn_">
         <property name="text">
          <string>↓</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="focus_down_btn_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="focus_label_">
           <property name="text">
            <string>远近</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="focus_up_btn_">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="aperture_down_btn_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="aperture_label_">
           <property name="text">
            <string>光圈</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="aperture_up_btn_">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>onCheckBoxStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>up_btn_</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>onUpPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>left_btn_</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>onLeftPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>down_btn_</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>onDownPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>right_btn_</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>onRightPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>left_btn_</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>onReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>up_btn_</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>onReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>right_btn_</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>onReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>down_btn_</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>onReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focus_down_btn_</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>onFocusDownPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focus_down_btn_</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>onReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focus_up_btn_</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>onFocusUpPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focus_up_btn_</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>onReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aperture_down_btn_</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>onApertureDownPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aperture_down_btn_</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>onReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aperture_up_btn_</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>onApertureUpPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aperture_up_btn_</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>onReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>call_presetpoint_btn_</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>onCallPresetPointClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_presetpoint_btn_</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>onSetPresetPointClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_presetpoint_btn_</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>onDeletePresetPointClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCheckBoxStateChanged(int)</slot>
  <slot>onUpPressed()</slot>
  <slot>onLeftPressed()</slot>
  <slot>onDownPressed()</slot>
  <slot>onRightPressed()</slot>
  <slot>onReleased()</slot>
  <slot>onFocusDownPressed()</slot>
  <slot>onFocusUpPressed()</slot>
  <slot>onApertureDownPressed()</slot>
  <slot>onApertureUpPressed()</slot>
  <slot>onCallPresetPointClicked()</slot>
  <slot>onSetPresetPointClicked()</slot>
  <slot>onDeletePresetPointClicked()</slot>
 </slots>
</ui>
